src/protocol/fluvio-protocol-api/src/request.rs

```diff
@@ -132,6 +135,7 @@ impl<R> Decoder for RequestMessage<R>
 where
     R: Request,
 {
+    #[instrument(level = "trace", skip(self, src, version))]
     fn decode<T>(&mut self, src: &mut T, version: Version) -> Result<(), IoError>
```

Comment:
These are very low level call, it just add lots of noise, is there any benefit to add tracing?